Mathlib/RingTheory/WittVector/Basic.lean

```diff
@@ -256,8 +251,6 @@ private def ghostEquiv' [Invertible (p : R)] : 𝕎 R ≃ (ℕ → R) where
 
 @[local instance]
 private def comm_ring_aux₁ : CommRing (𝕎 (MvPolynomial R ℚ)) :=
-  -- Porting note: no longer needed?
-  -- letI : CommRing (MvPolynomial R ℚ) := MvPolynomial.commRing
```

Comment:
I think this requires no action. The required instance is now called `MvPolynomial.instCommRingMvPolynomial`
and it is synthesised fine:

```
[Meta.synthInstance] ✅ CommRing (MvPolynomial R ℚ) ▼
  [] new goal CommRing (MvPolynomial R ℚ) ▶
  [] ❌ apply inst✝ to CommRing (MvPolynomial R ℚ) ▶
  [] ❌ apply inst✝¹ to CommRing (MvPolynomial R ℚ) ▶
  [] ❌ apply inst✝² to CommRing (MvPolynomial R ℚ) ▶
  [] ✅ apply @MvPolynomial.instCommRingMvPolynomial to CommRing (MvPolynomial R ℚ) ▶
  [] result MvPolynomial.instCommRingMvPolynomial 
```